lib/vsc/install/shared_setup.py

```diff
@@ -155,7 +155,7 @@ def _log(self, level, msg, args):
 
 RELOAD_VSC_MODS = False
 
-VERSION = '0.11.6'
+VERSION = '0.11.7'
```

Comment:
Maybe this one deserves a bigger version bump, since this is going to make a lot of tests fail in the repositories where `vsc-install` is used to run the tests... :)

`0.13.0` is a nice version number for that ;)

Reply:
@boegel Done!